### PR TITLE
Changing Navbar into Material-UI

### PR DESCRIPTION
**Pull request by rak-shit**
Screenshot 2020-04-09 at 12 48 31 AM

### CODE REVIEW

File: src/user/dashboard/navigation/navigation.js [hunk 1]
```diff
@@ -19,146 +22,145 @@ class Navigation extends Component {
...
+          </Link>
+          <ListItem button style={divStyle} onClick={() => this.setState({logout: true})}>
+            <ListItemIcon>
+              <img class="logout" src={logo} alt="L"></img>
+            </ListItemIcon>
+           <ListItemText primary="Logout" />
+          </ListItem>
+          <Logout show={this.state.logout} onHide={cancel} />
+          <ListItem button className="jitsi-box" onClick={() => this.setState({open: true})}>
+            <ListItemText primary="Jitsi Meet" className="jitsi" />
```
Comment (Rupeshiya) [inline src/user/dashboard/navigation/navigation.js:160]: add this icon there import DuoIcon from '@material-ui/icons/Duo'; and also make the alignment better!

File: src/user/dashboard/navigation/navigation.js [hunk 2]
```diff
@@ -19,146 +22,145 @@ class Navigation extends Component {
...
+                >
+                  <path
+                    d="M0 10H8V0H0V10ZM0 18H8V12H0V18ZM10 18H18V8H10V18ZM10 0V6H18V0H10Z"
+                    fill="black"
+                    fill-opacity="0.5"
+                    className="path-name"
+                  />
+                </svg>
+              </ListItemIcon>
+              <span className="link-text">Dashboard</span>
```
Comment (Rupeshiya) [inline src/user/dashboard/navigation/navigation.js:61]: add the property text-decoration to null, as on hover giving the underline and also use the same font-style for all the options on the sidebar.
make settings and logout option sticky to that point.

File: src/user/dashboard/navigation/navigation.js [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
-import { ListGroup, Button } from "react-bootstrap";
-import { NavLink } from 'react-router-dom';
+import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { DonutTitleSmall } from "../../../donutTitle/donutTitle";
 import "./navigation.scss";
 import {Logout} from "../../profile/popups/logout";
 import logo from "../../../svgs/logout.svg";
 import {Info} from "../../integrations/NameForm";
+import List from '@material-ui/core/List';
```
Comment (Rupeshiya) [inline src/user/dashboard/navigation/navigation.js:9]: Use desctructuring here

Comment (rak-shit) [reply src/user/dashboard/navigation/navigation.js:9]: Yeah sure there are some small changes I must do. I thought of doing it when I make the final commit. I’ll do it now anyway😁

Comment (Rupeshiya) [reply src/user/dashboard/navigation/navigation.js:9]: yeah cool!

**Comment by theprover97** [comment]
@rak-shit @Rupeshiya don't we already have PR #369 for this?

**Comment by Rupeshiya** [comment]
> @rak-shit @Rupeshiya don't we already have PR #369 for this?

Oh yes, it's already there.
@rak-shit Please check once before sending any PR or opening an issue.
Sorry @theprover97 It wasn't in my mind.

**Pull request closed without merging**